app/gui/src/dashboard/providers/DriveProvider.tsx

```diff
@@ -41,8 +42,8 @@ interface DriveStore {
   readonly setCanDownload: (canDownload: boolean) => void
   readonly pasteData: PasteData<DrivePastePayload> | null
   readonly setPasteData: (pasteData: PasteData<DrivePastePayload> | null) => void
-  readonly expandedDirectoryIds: readonly DirectoryId[]
-  readonly setExpandedDirectoryIds: (selectedKeys: readonly DirectoryId[]) => void
+  readonly expandedDirectories: Record<Path, readonly DirectoryId[]>
```

Comment:
Btw, why is it `Path` instead of `CategoryId`?

Reply:
we want to expand parents of the current asset on *all* categories this asset appears in (so all categories whose path matches the path of the current asset)

Reply:
How many categories can we have with the same asset ?

Reply:
cloud: 2 (root + user or root + team)
local: unlimited (all parent directories of the asset, if multiple are favorited)

Reply:
AFAIR for local projects id ~~~ path, 
`Root` will be removed soon I think (no need to think about)

Reply:
*local categories? yeah, fair enough. so, what's the verdict? switch to `CategoryId` now?

Reply:
Yeah, if possible it's better to use ids instead